golos.vesting/golos.vesting.hpp

```diff
@@ -34,10 +34,25 @@ class vesting : public contract {
         const auto& balance = balances.get(sym.raw());
         return balance.vesting;
     };
-    inline asset get_account_effective_vesting(name account, symbol_code sym) const;
-    inline asset get_account_available_vesting(name account, symbol_code sym) const;
-    inline asset get_account_unlocked_vesting(name account, symbol_code sym) const;
-    inline bool balance_exist(name owner, symbol_code sym) const;
+    static inline asset get_account_effective_vesting(name code, name account, symbol_code sym) {
+        tables::account_table balances(code, account.value);
+        const auto& balance = balances.get(sym.raw());
+        return balance.effective_vesting();
+    }
+    static inline asset get_account_available_vesting(name account, symbol_code sym) {
+        tables::account_table balances(_self, account.value);
```

Comment:
`_self` in static? : )
@DedbaradeD, change to `code` please. `code` must be provided as function argument